MapboxMobileEvents/MMEAPIClient.m

```diff
@@ -75,6 +77,11 @@ - (void)postEvents:(NSArray *)events completionHandler:(nullable void (^)(NSErro
             if (completionHandler) {
                 error = error ?: statusError;
                 completionHandler(error);
+              
```

Comment:
it looks like we're violating DRY with this block of code. should we consider wrapping it with helper method?

Reply:
Looks like this still needs to be addressed?

Reply:
Yeah, sorry. I had misinterpreted this to be referring to something else. Should be fixed now